File: C#/Practice/LeetCode/Easy/Sum of Unique Elements.cs
// https://leetcode.com/problems/sum-of-unique-elements/description/

public class Solution {
    public int SumOfUnique(int[] nums) {
        int sum = 0;

        for (int i = 0; i < nums.Length; i++) {
            if (Array.IndexOf(nums, nums[i]) == Array.LastIndexOf(nums, nums[i])) {
                sum += nums[i];
            }
        }

        return sum;        
    }
}

File: TypeScript/Practice/LeetCode/Easy/Sum of Unique Elements.ts
// https://leetcode.com/problems/sum-of-unique-elements/description/

function sumOfUnique(nums: number[]): number {
    let sum: number = 0;

    for (let i = 0; i < nums.length; i++) {
        if (nums.indexOf(nums[i]) === nums.lastIndexOf(nums[i])) {
            sum += nums[i];
        }
    }

    return sum;    
};